### DISCUSSION TITLE
v17.1.0

### DISCUSSION THREAD

**Discussion by carloscabanero** [blinksh/blink#1884] [Announcements]
We continue with our focus set on the Terminal, Mosh and SSH until the end of year. Here are some fixes:

- Create new host by duplicating another (Thanks to Krathen for the heads up) #1108
- Unified the saving behavior for Hosts and User screens. Those required the user to press Save explicitly. We now make it more obvious with Save and Cancel  #1742
- Some servers, like GitHub, would emit a message on stderr and disconnect. We are now showing the error in that case. #1448
- Deleting a Host would not immediately make it disappear from the Files.app until restart. We now sync the FP properly.  #1857
- #1888
- #1874

**Comment by s3ase** [comment]
Can you help me understand the statement "We continue with our focus set on the Terminal, Mosh and SSH until the end of year" especially the last part "until the end of year" will Blink be changing next year?

**Reply by carloscabanero** [reply]
No changes planned. Just meant that our focus was on fixing bugs and polishing that part of the app. A lot of the work we did on those months was setting up the foundation for the next releases too.
